tests/unit/assert.test.ts

```diff
@@ -0,0 +1,80 @@
+import assert from "assert";
+
+describe("assert.ok", () => {
+  it("Should be returned 'undefined' (So it's not an error)", () => {
+    expect(assert.ok(true)).toBeUndefined(); //bool
+    expect(assert.ok(1)).toBeUndefined(); // numeric
+    expect(assert.ok("non-empty string")).toBeUndefined(); // string
+    expect(assert.ok([])).toBeUndefined(); // array
+    expect(assert.ok({})).toBeUndefined(); // object
+    expect(assert.ok(() => {})).toBeUndefined(); // function
+    expect(assert.ok(123n)).toBeUndefined(); // bigint
+    expect(assert.ok(Symbol())).toBeUndefined(); // symbol
+    expect(assert.ok(new Error())).toBeUndefined(); // error
+    class AssertTestClass {}
+    expect(assert.ok(AssertTestClass)).toBeUndefined(); // constructor
+  });
+
+  it("Should be returned exception", () => {
+    const errMsg =
+      "AssertionError: The expression was evaluated to a falsy value";
+    try {
+      assert.ok(false);
+    } catch (err) {
+      expect(err.message).toEqual(errMsg);
+    }
+    try {
+      assert.ok(0);
+    } catch (err) {
+      expect(err.message).toEqual(errMsg);
+    }
+    try {
+      assert.ok("");
+    } catch (err) {
+      expect(err.message).toEqual(errMsg);
+    }
+    try {
+      assert.ok(null);
+    } catch (err) {
+      expect(err.message).toEqual(errMsg);
+    }
```

Comment:
You should use expect to detect throws and not try catch:
expect(t).toThrow("message");